include/wlr/types/wlr_box.h

```diff
@@ -32,4 +34,20 @@ void wlr_box_transform(const struct wlr_box *box,
 void wlr_box_rotated_bounds(const struct wlr_box *box, float rotation,
 	struct wlr_box *dest);
 
+void wlr_box_from_pixman_box32(const pixman_box32_t box, struct wlr_box *dest);
+
+static const struct wlr_box WLR_BOX_MAX = {
```

Comment:
This still doesn't work

Reply:
acrisci's comment about it was hidden after I updated the file, so I forgot about it. I'll fix this ASAP.

Reply:
I actually can't find it anymore, I'll ask him again about the implications.

Reply:
You cannot construct a BOX_MAX because this box has INT_MAX * 2 width and height. Output layout coordinates are allowed to be negative and it will not intersect with those.

Reply:
I wonder what I should do instead. I suppose I'll get rid of the WLR_BOX_MAX concept then completely.